sway-lsp/src/core/session.rs

```diff
@@ -328,3 +330,59 @@ impl Session {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::env;
+
+    use super::*;
+
+    fn sway_workspace_dir() -> PathBuf {
```

Comment:
Perhaps it's worth making a `test_utils.rs` file under the new tests folder that contains useful functions like this and  `get_absolute_path` below. Then we don't need to duplicate them in other tests such as in the `document.rs` file above.

Reply:
I was thinking this too, will do.